plugins/cpu/cpu_osx

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+# -*- sh -*-
+
+# shellcheck disable=SC2046
+
+: << =cut
+
+=head1 NAME
+
+cpu - Plugin to measure cpu on osx.
+
+=head1 NOTES
+
+This plugin runs the top command once per interval, to discover the current value of CPU usage on OSX.
+The result is scaled to # of CPU's, so a 4 core machine will reach 400% utilization.
+Contributions are welcome to convert the plugin to use a long running counter such as /proc/stat in Linux.
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+ #%# family=auto
+ #%# capabilities=autoconf
+
+=cut
+
+NCPU=$(sysctl hw.ncpu | cut -d" " -f2)
+
+if [ "$1" = "autoconf" ]; then
```

Comment:
The `autoconf` capability is optional. Since it does nothing here, you should remove it.